EIPS/eip-txindex-opcode.md

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
```suggestion
---
eip: 7793
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.